docs/wiki/CL/cl-clients.md

```diff
@@ -1,16 +1,18 @@
+<!-- @format -->
```

Comment:
I think your IDE creates this tag in every doc? Please leave it out from the final page.

Reply:
removed